### DISCUSSION TITLE
RSS of repositories of an author

### DISCUSSION THREAD

**Discussion by slhuilli1** [community/community#80989] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello,

I am looking for a RSS feed of all my repositories in order to show it on my website. Is there an URL for that ? Thanks

**Comment by meralhamarat** [comment, marked as answer]
Hello
As far as I know, GitHub currently does not provide an automatic RSS feed for all repositories of users

**Reply by slhuilli1** [reply]
thanks a lot

**Reply by meralhamarat** [reply]
Your welcome :)
